src/records/ConsoleEntry.js

```diff
@@ -4,4 +4,5 @@ export default Record({
   expression: '',
   value: null,
   error: null,
+  projectKey: '',
```

Comment:
I would probably call this `evaluatedByCompiledProjectKey` and set its default to `null`